extensions/patterns/configure_ec2/template_rhdh/create_ec2_instance.yml

```
@@ -0,0 +1,180 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: cloud-configure-ec2
+  title: AWS Operations / Create EC2 Instance
+  description: This wizard will guide you on how to create EC2 instance in the Amazon cloud
+  namespace: default
+  tags:
+    - aap-operations
+    - intermediate
+    - cloud-aws-ops
+    - configure-ec2-pattern
+    - create-ec2-instance
+spec:
+  type: service
+  parameters:
+    - title: Prompts
+      description: Create EC2 instance in the Amazon cloud
+      required:
+        - token
+        - inventory
+      properties:
+        token:
+          title: Token
+          type: string
+          description: Oauth2 token
+          ui:field: AAPTokenField
+          ui:widget: password
+          ui:backstage:
+            review:
+              show: false
+        inventory:
+          title: Inventory
+          description: Please enter the inventory you want to use the services on
+          resource: inventories
+          ui:field: AAPResourcePicker
+        credentials:
+          title: Credentials
+          description: Select credentials for accessing AWS.
+            The credentials need to be of type "Amazon Web Services".
+          type: array
+          ui:field: AAPResourcePicker
+          resource: credentials
+        verbosity:
+          title: Verbosity
+          description: Control the level of output Ansible will produce as the playbook executes.
+          resource: verbosity
+          ui:field: AAPResourcePicker
+    - title: Survey
+      required:
+        - aws_region
+        - instance_name
+        - instance_type
+      description: Create EC2 instance in the Amazon cloud
+      properties:
+        aws_region:
+          title: AWS Region
+          description: AWS region where resources should be created
+          type: string
+          ui:options:
+            rows: 5
+        instance_name:
+          title: Instance Name
+          description: Name of EC2 instance to create
+          type: string
+          ui:options:
+            rows: 5
+        instance_type:
+          title: Instance Type
+          description: Type of EC2 instance to create (e.g., t2.micro, m5.large)
+          type: string
+          ui:options:
+            rows: 5
+        ami_id:
+          title: AMI ID
+          description: Amazon Machine Image (AMI) ID to use for the instance, if not provided will default to the RHEL 9 AMI for the provided region and instance type
+          type: string
+          ui:options:
+            rows: 5
+        key_name:
+          title: Key Pair Name
+          description: Name of key pair to use or create for SSH access to the EC2 instance. Defaults to '{{ instance_name }}-key'
```

```suggestion
          description: Name of key pair to use for SSH access to the EC2 instance. If the key does not exist or is not provided, the instance will not be accessible via SSH.
```

We updated this in the survey template and it should be updated here as well. Thanks!